simple_plot.py

```diff
@@ -313,11 +333,22 @@ def plotData(NQuery, input_table, FigureStrBase, variables, xMin, xMax,
         ax.text(xf + 0.05, yf-0.01, str(10**rad) + ' ' + str(zMin.unit),
                 transform=ax.transAxes)
 
+    # Saving the plots
+
+    if html_dir is None:
+        html_dir = "/"
+
+    if png_dir is None:
+        png_dir = "/"
```

Comment:
these could just be "" since you're stripping and adding on the next line, but I suppose it doesn't matter